localgov.info.yml

```diff
@@ -35,12 +35,14 @@ install:
   - admin_toolbar:admin_toolbar
   - admin_toolbar:admin_toolbar_tools
   - entity_browser:entity_browser
+  - dropzonejs:dropzonejs
```

Comment:
This line isn't needed as simple_media_bulk_upload already has a dependency on this module.